### DISCUSSION TITLE
[Suggestion] Using proot

### DISCUSSION THREAD

**Discussion by aroxu** [vhqtvn/VHSCode-Android#154] [Ideas]
run code-server inside of proot distro.
I think it can provide more packages and languages such as C#, dotnet

**Comment by vhqtvn** [comment]
I'm current have too little time for this project :( I'll try working on this when i have time. PRs are always welcome!

**Comment by wilgnne** [comment]
@vhqtvn is there any support material that can help in this implementation?

**Comment by vhqtvn** [comment]
As i mirrored all the packages from termux, this can be implemented the exact way as termux, but in that case within proot it's a whole os that you can install code-server easily so maybe that would be a project like DevOS not VHEditor anymore...
